tests/PhpSpreadsheetTests/WorksheetTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheetTests;
+
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+use PhpOffice\PhpSpreadsheet\Worksheet;
+use PHPUnit_Framework_TestCase;
+
+class WorksheetTest extends PHPUnit_Framework_TestCase
+{
+    public function testSetTitle()
+    {
+        $test_title = str_repeat('a', 31);
+
+        $worksheet = new Worksheet();
+        $worksheet->setTitle($test_title);
+        $this->assertSame($test_title, $worksheet->getTitle());
+    }
+
+    public function setTitleInvalidProvider()
+    {
+        return [
+            [str_repeat('a', 32), 'Maximum 31 characters allowed in sheet title.'],
+            ['invalid*title', 'Invalid character found in sheet title'],
+        ];
+    }
+
+    /** @dataProvider setTitleInvalidProvider */
```

Comment:
It seems code style is [choking on this](https://travis-ci.org/PHPOffice/PhpSpreadsheet/jobs/252118853#L484). Could you make it a standard docblock please ? Then I'll merge it.